app/pb_kits/playbook/plugins/pb_chart.js

```diff
@@ -30,33 +55,57 @@ class pbChart {
     this.settings = this.extendDefaults(this.defaults, options)
 
     if (this.options.type == 'variablepie' || this.options.type ==  'pie'){
-      this.setupPieChart()
+      this.setupPieChart(options)
     } else {
       this.setupChart()
     }
   }
 
-  setupPieChart() {
+  setupPieChart(options) {
     Highcharts.setOptions(highchartsTheme)
-
     Highcharts.chart(this.defaults.id, {
       title: {
         text: this.defaults.title,
       },
       chart: {
         type: this.defaults.type,
+        events: {
+          render: function(event) {
+            const itemToMove = document.querySelector(`#wrapper-circle-chart-${event.target.renderTo.id} .pb_circle_chart_block`)
+            const chartContainer = document.querySelector(`#${event.target.renderTo.id}`)
+            if (itemToMove !== null) {
+              itemToMove.style.height = `${event.target.chartHeight}px`
+              itemToMove.style.width = `${event.target.chartWidth}px`;
+              (chartContainer.firstChild).before(itemToMove)
+            }
+          },
```

Comment:
We can move it to a separate function for sure. I wouldn't want to move  it outside of this file however because it would separate the two areas too much in my view. My two cents.